pom.xml

```diff
@@ -1,95 +1,112 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.google.cloud.solutions.flexenv</groupId>
-  <artifactId>backend</artifactId>
-  <packaging>war</packaging>
-  <version>0.0.1-SNAPSHOT</version>
-  <name>App Engine Backend module for Firebase</name>
-  <url>https://cloud.google.com</url>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```

Comment:
What is the reason for the change in indentation?

Reply:
That's reformatted by my IDE so they have consistent indentations with the rest of the file.

Reply:
Mmm... changes should not be the result of your IDE configuration. It makes it difficult to identify the changes that you are actually proposing. Additionally, the next person to propose a change might have a different configuration.
May I suggest that you configure your IDE with 2-space indentation, per https://google.github.io/styleguide/xmlstyle.html?